src/GitHub.Exports/Primitives/UriString.cs

```diff
@@ -197,7 +166,7 @@ public override UriString Combine(string addition)
             if (url != null)
             {
                 var urlBuilder = new UriBuilder(url);
-                if (!String.IsNullOrEmpty(urlBuilder.Query))
+                if (!string.IsNullOrEmpty(urlBuilder.Query))
```

Comment:
Nope. I think it's my overeager R# that does this. We should pick an approach and stick to it. Do you have a preference?

BTW, this has nothing to do with value type or class type. `string` is just the shorthand for `System.String` but it's the exact same class. :smile: